Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -493,12 +490,10 @@ const u8* JitArm64::DoJit(u32 em_address, PPCAnalyst::CodeBuffer* code_buf, JitB
     js.instructionsLeft = (code_block.m_num_instructions - 1) - i;
     const GekkoOPInfo* opinfo = ops[i].opinfo;
     js.downcountAmount += opinfo->numCycles;
+    js.isLastInstruction = i == (code_block.m_num_instructions - 1);
 
-    if (i == (code_block.m_num_instructions - 1))
-    {
-      // WARNING - cmp->branch merging will screw this up.
```

Comment:
I don't want to implement this kind of instruction merging, so IMO this comment isn't valid.